src/applications/personalization/profile-2/tests/components/ProfileWrapper.unit.spec.js

```diff
@@ -209,9 +223,13 @@ describe('mapStateToProps', () => {
         expect(props.showLoader).to.be.false;
       });
 
-      it('is `false` when all required data calls have either resolved successfully or errored', () => {
+      it('is `false` when all required data calls have errored', () => {
         const state = makeDefaultState();
-        state.vaProfile.paymentInformation.error = {};
```

Comment:
this was actually a bad test. we really wanted to totally replace the `paymentInformation` data with just error data vs. just tacking the error onto the resolved data that's part of the `makeDefaultState()`